### ISSUE TITLE
Collapse many consecutive filtered messages

### ISSUE THREAD

**Issue by rock3r** [pachli/pachli-android#396]
**Is your feature request related to a problem? Please describe.**
When there are many filtered messages in s row, the timeline gets filled with them;

![Screenshot_20240130-120714](https://github.com/pachli/pachli-android/assets/153802/90879904-677a-458f-b90f-5172c645d411)

**Describe the solution you'd like**
I'd love it if it collapsed many consecutive filtered messages for the same filter into a "X messages filtered: {filter value}"

Thanks! :)

**Comment by nikclayton** [comment]
Thanks for the suggestion.

How would that work if you only wanted to show some but not all of the messages? With the current approach you've got a choice. If they were collapsed to one you'd have a single "SHOW ANYWAY" button that would show all of them.

**Comment by rock3r** [comment]
In my case, for example, I'm filtering out long threads, since Pachli doesn't collapse them. I'd be more than happy to show all consecutive filtered out messages with one button. Or with collapsing long threads! Either way works :)

**Comment by nikclayton** [comment]
I see.

So actually filtering out the messages in the thread apart from the first is what you're looking for?

Basically, filter out any message that's a reply if the message it's replying to is from the same author?

**Comment by rock3r** [comment]
Yeah, that's the main case when I see these sort of long lists of consecutive filtered messages
